app/recurring_job.go

```diff
@@ -785,6 +791,20 @@ func (job *Job) UpdateVolumeStatus(v *longhorn.Volume) (*longhorn.Volume, error)
 	return job.lhClient.LonghornV1beta2().Volumes(job.namespace).UpdateStatus(context.TODO(), v, metav1.UpdateOptions{})
 }
 
+// GetSettingAsBool returns boolean of the setting value searching by name.
+func (job *Job) GetSettingAsBool(name string) (bool, error) {
+	obj, err := job.lhClient.LonghornV1beta2().Settings(job.namespace).Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
nit: Consider using `types.SettingName` to prevent rewriting the conversion from the caller each time this method is reused.
```suggestion
func (job *Job) GetSettingAsBool(name types.SettingName) (bool, error) {
	obj, err := job.lhClient.LonghornV1beta2().Settings(job.namespace).Get(context.TODO(), string(name), metav1.GetOptions{})
```

Reply:
Modified. Thanks